### ISSUE TITLE
Releases to NPM

### ISSUE THREAD

**Issue by korverdev** [gnat/surreal#44]
I'd love to see this package made available on NPM. Unfortunately, my company is very particular about always downloading dependencies off NPM and outright bans other sources, so if this was possible, it would help a lot for to drive adoption on my team.

**Comment by korverdev** [comment]
[There's at least one person who I don't believe is affiliated with this project who's taken it on themselves to publish this project](https://www.npmjs.com/package/surreal.me), but it's fallen out of date and I would have security concerns using an NPM package that isn't endorsed by the people behind the library.

**Comment by stag-enterprises** [comment]
that's me btw :)

i haven't used it recently so i really never even realized it was out of date 🫠

**Comment by gnat** [comment]
First party distribution on NPM is planned for the next release, assuming no gotchas. Thanks for trying out the project!

**Comment by frederikb** [comment]
Thanks, @gnat! Looking forward to the release.

This will also enable us to consume the library via [WebJars](https://www.webjars.org) in the Java ecosystem. WebJars automatically bundles up NPM dependencies in a format consumable via the dependency management used in that ecosystem.

tl;dr: Happy user

**Comment by gnat** [comment]
Official automated NPM releases are now available at https://www.npmjs.com/package/@geenat/surreal

Enjoy !!

**Comment by frederikb** [comment]
Thanks @gnat, appreciated!

You might want to add that information to the respective section in the `README.md` as well?

Also, the published NPM package still contains some unnecessary files in my opinion.

<img width="1582" height="440" alt="Image" src="https://github.com/user-attachments/assets/a26ac0aa-9b4e-42d3-a980-a56d4ceed848" />

You should be able to exclude those by switching to a whitelist approach in the `package.json` of including only certain files:

```json
{
  // …
  "files": [
    "surreal.js",
    "README.md",
    "LICENSE"
  ]
}
```

**Comment by gnat** [comment]
Done @frederikb thank you for the great suggestion!
